### ISSUE TITLE
Tinker's books are incompatible with Akashic Tome

### ISSUE THREAD

**Issue by MuteTiefling** [NillerMedDild/Enigmatica6#2351]
**Modpack version**
0.5.0 dev

**Describe the bug**
None of the tinker's books can be added to the akashic tome

https://github.com/SlimeKnights/TinkersConstruct/issues/4395

**Comment by NillerMedDild** [comment]
We usually have to add them to the config, before Akashic Tome accepts them. It would be nice if the Tome just accepted things with the tag `forge:books/guide` though.

**Comment by MuteTiefling** [comment]
Hrm, ok wasn't aware of that. I've added them to the Akashic Tome whitelist in my test instance but they're still not going in :D

**Comment by dawidmachon** [comment]
Any progress on that?

**Comment by MuteTiefling** [comment]
We can't force what a mod won't accept.

**Comment by theboo** [comment]
wasnt this fixed ages ago and we have the big ol' book in the tome now? just not the 4 other ones?

**Comment by MuteTiefling** [comment]
yeah, we have the encyclopedia, but it only allows one book. So that's all we can put, instead of all 5
